openshift_tools/monitoring/awsutil.py

```diff
@@ -62,10 +62,10 @@ def _run_cmd(self, cmd):
 
         return results.stdout.read()
 
-    def get_bucket_list(self, verbose=False):
+    def get_bucket_list(self, verbose=False, BucketRegion='eu-central-1'):
```

Comment:
I suggest:

```
BucketRegion=None
if not BucketRegion:
    BucketRegion = " --region " + BucketRegion
```

As this is a library function we do not want to change the default functionality.